packages/apps-config/src/ui/general/index.ts

```diff
@@ -37,6 +37,14 @@ const nodeColors: Record<string, any> = [
   [node.toLowerCase().replace(/-/g, ' ')]: color
 }), {});
 
+function sanitize (value?: string): string {
+  return value?.toLowerCase().replace('-', ' ') || '';
+}
+
+export const getSystemChainColor = (systemChain: string, systemName: string): string | undefined => {
```

Comment:
1. Move to ui/index.ts (alongside other utility)
2. use `export function (...): string | undefined { ... }`

Reply:
which `ui/index.ts` file, could you please name the full path? in `apps`?

Reply:
Here, there is a util in there already - https://github.com/polkadot-js/apps/blob/master/packages/apps-config/src/ui/index.ts#L7

Reply:
thanks, I see it, just updated with `master`